go.mod

```diff
@@ -2,43 +2,48 @@ module github.com/smartcontractkit/ocr2keepers
 
 go 1.19
 
-replace github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.23.3
+replace (
+	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.23.3
+	golang/github.com/gogo/protobuf => golang/github.com/gogo/protobuf v1.3.3
+)
 
 require (
 	github.com/Maldris/mathparse v0.0.0-20170508133428-f0d009a7a773
-	github.com/ethereum/go-ethereum v1.10.17
-	github.com/go-echarts/go-echarts/v2 v2.2.4
-	github.com/google/uuid v1.2.0
+	github.com/ethereum/go-ethereum v1.10.26
+	github.com/go-echarts/go-echarts/v2 v2.2.5
+	github.com/google/uuid v1.3.0
+	github.com/pkg/errors v0.9.1
 	github.com/shopspring/decimal v1.3.1
-	github.com/smartcontractkit/libocr v0.0.0-20221209172631-568a30f68407
-	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/testify v1.7.2
-	go.uber.org/multierr v1.6.0
-	golang.org/x/crypto v0.0.0-20221012134737-56aed061732a
+	github.com/smartcontractkit/libocr v0.0.0-20230112103111-f100435787c8
```

Comment:
what is the version used in CL repo?

Reply:
The chainlink node uses commit 568a30f but this repo uses the latest commit of f100435.

Reply:
So not use the latest?

Reply:
yep

Reply:
done